package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-cli-autoprefixer",
-  "version": "0.7.0",
+  "version": "0.8.0",
```

Comment:
In the future, please don't bump the version number of the package in the PR ✨

Reply:
I had left this PR open to allow edits from maintainers, no need to make your own PR. Thanks for the merge ✨

Reply:
Ok, noted. Thanks for the merge!